tox.ini

```diff
@@ -51,10 +53,17 @@ deps =
     {alldeps,tests,coverage}: priority
     {alldeps,tests,coverage}: sphinx
 
-    py27-{alldeps,tests,coverage}-posix: pysqlite
+    py27-{alldeps,tests,coverage}-{posix,macos}: pysqlite
     py27-{alldeps,tests,coverage}: soappy
 
-    alldeps-windows: pypiwin32
+    windows: pypiwin32
+
+    {alldeps,tests}-macos: pyobjc-core
+    {alldeps,tests}-macos: pyobjc-framework-cfnetwork
```

Comment:
Also, canonical capitalization here is `"CFNetwork"`, as per https://pypi.python.org/pypi/pyobjc-framework-CFNetwork